src/Utils/obiDialogParser.ts

```diff
@@ -6,6 +6,7 @@ import * as CLM from '@conversationlearner/models'
 import * as DialogEditing from './dialogEditing'
 import * as OBIUtils from './obiUtils'
 import * as Util from './util'
+import * as stripJson from 'strip-json-comments'
```

Comment:
Why did you remove the "comments" part of the import?

`stripJson` now it sounds like it's stripping out json.... from json.  
if you want to shorten name call it `stripComments`